WooCommerce/WooCommerceUITests/README.md

```diff
@@ -67,7 +67,7 @@ The following flows are covered/planned to be covered by UI tests. Tests that ar
     - [x] Universal Link to Payments screen
     - [x] Universal Link to an Order
 8. Settings
-    - [ ] Contact support
+    - [x] Contact support
```

Comment:
I did not review the PR in detail today, unfortunately, but I'd like to raise a point about how and when we consider a test automated. Given we use mocks, automating a single test from critical flows / checklist might mean automating it on several layers: UI, API, Unit. 

If we mark the `Contact support` test as automated, the changes it won't be executed manually any longer are high. However, with this test, only the UI layer is automated, which does not include the main part: making sure the support can really be contacted. Are we sure we have enough tests, on other levels, to consider it automated?

Reply:
yup that's a fair question, thanks for asking! even manually right now, i don't send a support ticket and validate that it's available on Zendesk on every release (i don't believe that would be very helpful for our HEs) and also, i think after tapping on the submit button, we're not testing our app anymore but testing zendesk API. 

what this test is really looking for is that the screen can be opened and the buttons/text fields that are made available can be interacted with. that said, i can see how it can be seen as incorrect to say that `contact support` is now automated. how about a middle ground where we expand `Contact support` to these two:
1. Contact support - validate buttons and text fields
2. Contact support - submit support ticket

so this PR covers (1), i'm not sure if we would want to automate (2) now (unless there's a way to clean up the tickets created from each PR commit. i vaguely remember a similar conversation in another repo...). wdyt?

Reply:
> i don't send a support ticket and validate that it's available on Zendesk on every release (i don't believe that would be very helpful for our HEs)

In recent times, I was guilty of actually sending the messages, but specifying that it's a part of a beta test and it can be ignored. Not sure how hurtful this is to HE, but I'm doing less of this now.

> how about a middle ground where we expand Contact support to these two

I think it's a great solution, because it clearly communicates what is covered by a UI tests, and what is not covered 👍